troposphere/codedeploy.py

```diff
@@ -164,6 +173,30 @@ class TriggerConfig(AWSProperty):
     }
 
 
+class Ec2TagGroup(AWSProperty):
+    props = {
+        'Ec2TargetGroup': ([Ec2TagFilters], False)
+    }
+
+
+class Ec2TagSet(AWSProperty):
+    props = {
+        'Ec2TagSet': ([Ec2TagGroup], False)
+    }
+
+
+class OnPremisesTagGroup(AWSProperty):
+    props = {
+        'OnPremisesTagGroup': ([TagFilters], False)
+    }
+
+
+class OnPremisesTagSet(AWSProperty):
+    props = {
+        'OnPremisesTagSet': ([OnPremisesTagGroup], False)
```

Comment:
This property should be `OnPremisesTagSetList` it looks like:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-codedeploy-deploymentgroup-onpremisestagset.html

It looks like that's supposed to be a list of  `OnPremisesTagSetListObject` objects:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-codedeploy-deploymentgroup-onpremisestagsetlistobject.html